### ISSUE TITLE
[Bug]: Unexpected crash on HTMLInputElement.click() call

### ISSUE THREAD

**Issue by nosferatu500** [electron/electron#31871]
### Preflight Checklist

- [X] I have read the [Contributing Guidelines](https://github.com/electron/electron/blob/main/CONTRIBUTING.md) for this project.
- [X] I agree to follow the [Code of Conduct](https://github.com/electron/electron/blob/main/CODE_OF_CONDUCT.md) that this project adheres to.
- [X] I have searched the [issue tracker](https://www.github.com/electron/electron/issues) for a feature request that matches the one I want to file, without success.

### Electron Version

16.0.0

### What operating system are you using?

macOS

### Operating System Version

macOS Monterey 12.0.1

### What arch are you using?

x64

### Last Known Working Electron version

15.3.2

### Expected Behavior

Everything should work the same as in the previous (15.3.2) build.

### Actual Behavior

After updating on build 16.0, my app closes unexpectedly after calling ```inputRef.current.click();```.
Check example code please.

### Additional Information

Code example:
```
export const FilePicker: React.FC<Props> = (props) => {
    const inputRef = useRef<HTMLInputElement | null>(null);
    const [fileName, setFileName] = useState(props.value);

    return (
        <Input.Group>
            <Input value={fileName}  />
            <input
                ref={inputRef}
                style={{ display: "none" }}
                type="file"
                accept={props.accept}
                multiple={props.multiple}
                onChange={(event) => {
                    // do something
                }}
            />
            <Button
                onClick={() => {
                    if (inputRef.current) {
                         // BUG: Crash on call (but everything fine on build 15.3.2)         <- here
                         inputRef.current.click();                   
                    }
                }}
            >
                ...
            </Button>
        </Input.Group>
    );
};
```

**Comment by nosferatu500** [comment]
UPD: Error in console
```
Electron exited with signal SIGTRAP
```

**Comment by codebytere** [comment]
Thanks for reporting this and helping to make Electron better!

Because of time constraints, triaging code with third-party dependencies is usually not feasible for a small team like Electron's.

Would it be possible for you to make a standalone testcase with only the code necessary to reproduce the issue? For example, [Electron Fiddle](https://www.electronjs.org/fiddle) is a great tool for making small test cases and makes it easy to publish your test case to a [gist](https://gist.github.com) that Electron maintainers can use.

Stand-alone test cases make fixing issues go more smoothly: it ensure everyone's looking at the same issue, it removes all unnecessary variables from the equation, and it can also provide the basis for automated regression tests.

I'm adding the `blocked/need-repro` label for this reason. After you make a test case, please link to it in a followup comment. This issue will be closed in 10 days if the above is not addressed.

**Comment by nosferatu500** [comment]
Hey! It looks like this has been fixed in build 16.0.1. Thanks!

**Comment by codebytere** [comment]
@nosferatu500 rad, glad to hear it :D
